### PR TITLE
Additions

### PR DESCRIPTION
**Pull request by joshmcintyre**
Batch update of site additions. I also closed 2 issues for sites that shouldn't be added in the process.

### CODE REVIEW

File: _data/adult.yml [hunk 1]
```diff
@@ -137,6 +137,19 @@ websites:
       bch: No
       othercrypto: No
 
+    - name: SexRoulette
+      url: https://telegram.me/SexRouletteBot_bot
+      img: sexroulette.png
+      twitter:
```
Comment (EchterAgo) [inline _data/adult.yml]: Remove empty attributes

File: _data/adult.yml [hunk 2]
```diff
@@ -137,6 +137,19 @@ websites:
       bch: No
       othercrypto: No
 
+    - name: SexRoulette
+      url: https://telegram.me/SexRouletteBot_bot
+      img: sexroulette.png
+      twitter: 
+      facebook: 
+      region: Global
```
Comment (EchterAgo) [inline _data/adult.yml]: Remove region/country/city, that basically means global

File: _data/adult.yml [hunk 3]
```diff
@@ -137,6 +137,19 @@ websites:
...
+      country: Global
+      city: Global
+      bch: yes
+      btc: no
+      othercrypto: no
+      doc:
```
Comment (EchterAgo) [inline _data/adult.yml]: Also remove because empty

File: _data/developer.yml [hunk 4]
```diff
@@ -423,6 +423,18 @@ websites:
...
+- name: Underlabs Inc.
+  url: https://underlabs.ca
+  img: underlabs.png
+  twitter: underlabs
+  email_address: hello@underlabs.ca
+  region: na
+  bch: Yes
+  btc: Yes
+  othercrypto: No
+  doc: https://underlabs.ca/smart-contracts.html
```
Comment (EchterAgo) [inline _data/developer.yml]: That is not a good doc link as it does not confirm if this service is accepting BCH

File: _data/developer.yml [hunk 5]
```diff
@@ -423,6 +423,18 @@ websites:
   btc: true
   othercrypto: true
   doc: https://ultimez.com/blog/ultimez-now-accepts-cryptocurrency-payment-web-services/
+- name: Underlabs Inc.
+  url: https://underlabs.ca
+  img: underlabs.png
+  twitter: underlabs
+  email_address: hello@underlabs.ca
+  region: na
```
Comment (EchterAgo) [inline _data/developer.yml]: Are they really only developing for clients from North America? If they are developing for any client, remove the tag.

File: _data/finance.yml [hunk 6]
```diff
@@ -274,6 +284,19 @@ websites:
       sms: Yes
       doc: https://www.taxact.com/support/24138/2016/how-is-a-verification-code-used
 
+    - name: Tradingview
+      url: https://www.tradingview.com/
+      img: tradingview.png
+      twitter: @tradingview
+      facebook: @tradingview
+      region:
```
Comment (EchterAgo) [inline _data/finance.yml]: Remove the empty attributes

File: _data/retail.yml [hunk 7]
```diff
@@ -49,6 +49,15 @@ websites:
   btc: true
   othercrypto: true
   doc: https://9figures.co.uk/blogs/news/accepting-cryptocurrency-1
+- name: AbrahamArtist
+  url: https://abrahamartist.myshopify.com/
+  img: abrahamartist.png
+  twitter: Mad_spinner
+  email_address: abrahama@abrahamartist.com
+  region: na
```
Comment (EchterAgo) [inline _data/retail.yml]: Remove region, they offer international shipping

File: _data/retail.yml [hunk 8]
```diff
@@ -169,6 +178,16 @@ websites:
   keywords:
   - bitpay
   - bip70
+- name: Autodekor
+    url: https://cockpitdekor.com
```
Comment (EchterAgo) [inline _data/retail.yml]: Fix indentation, should be 2 spaces, this is making rake fail

File: _data/retail.yml [hunk 9]
```diff
@@ -169,6 +178,16 @@ websites:
...
+    img: autodekor.png
+    twitter: Fityourselfcouk
+    facebook: cockpitdekor/
```
Comment (EchterAgo) [inline _data/retail.yml]: Remove trailing slash

File: _data/retail.yml [hunk 10]
```diff
@@ -625,6 +644,16 @@ websites:
   keywords:
   - bitpay
   - bip70
+- name: Grow Kit
+    url: https://thegrowkit.ca/
```
Comment (EchterAgo) [inline _data/retail.yml]: Again, fix indentation

File: _data/services.yml [hunk 11]
```diff
@@ -16,6 +16,15 @@ websites:
   bch: true
   btc: true
   othercrypto: false
+- name: BCH Ad Spot
+    url: https://bchadspot.com
```
Comment (EchterAgo) [inline _data/services.yml]: ...

File: _data/services.yml [hunk 12]
```diff
@@ -16,6 +16,15 @@ websites:
...
+    img: bch_adspot.jpg
+    email_address: info@bchadspot.com
+    bch: Yes
+    btc: No
+    othercrypto: No
+    exceptions:
```
Comment (EchterAgo) [inline _data/services.yml]: change the exception to

```
keywords:
- coinpayments.net
```

File: _data/finance.yml [hunk 13]
```diff
@@ -274,6 +284,19 @@ websites:
       sms: Yes
       doc: https://www.taxact.com/support/24138/2016/how-is-a-verification-code-used
 
+    - name: Tradingview
+      url: https://www.tradingview.com/
+      img: tradingview.png
+      twitter: @tradingview
```
Comment (EchterAgo) [inline _data/finance.yml]: Remove the @ in front of Twitter and Facebook accounts.

**Comment by joshmcintyre** [comment]
Thanks @EchterAgo. Makes it a lot easier to figure out what is wrong and how to correct. Will get to this ASAP.

**Comment by EchterAgo** [comment]
You're welcome. Also try `bundle exec rake` if you have a ruby environment
setup, it will give you most of the info as well.

**Comment by kenman345** [comment]
Still need to decide what to do with finance.yml but otherwise this PR would be ready

**Comment by kenman345** [comment]
I decided to change up all the listings in finance.yml to have proper tags. I did not check any of them. I simply changed them all to `bch: No` and removed the TFA based tags. We still need to add finance to the sections.yml file if we want to have any of these listed on the site. We should at least double check some of the listings too.

**Comment by joshmcintyre** [comment]
Thanks for your help, kenman! I'm a little confused reading the end of the chain though; do I need to do anything else with the listings that were in finance?

**Comment by kenman345** [comment]
right now `finance.yml` is not mentioned from the `sections.yml` file. That means it does not appear as a section to users, and all that data in `finance.yml` is not being used. Adding listings to there means the listings will not appear on the site after this is merged.

To rectify this, we have two solutions to deliberate and decide:
1. We can verify the data for those other listings in `finance.yml` as they havent been verified before, and then add an entry to `sections.yml` to add in a `Finance` section. 
2. We move those new listings to a section that already is being used. We then proceed to handle Option 1 in a more timely manner and move those listings to other sections if appropriate and/or leave them there and add the section once complete with verification of data.

File: _data/finance.yml [hunk 14]
```diff
@@ -13,318 +13,290 @@ websites:
...
 
     - name: LevelUp
       url: https://www.thelevelup.com
       twitter: TheLevelUp
       facebook: LevelUpApp
       img: levelup.png
-      tfa: No
+      bch: No
+
+    - name: MyMillennialGuide.com
```
Comment (kenman345) [inline _data/finance.yml]: This is a service to me, so lets move it

File: _data/finance.yml [hunk 15]
```diff
@@ -13,318 +13,290 @@ websites:
...
     - name: Taxact
       url: https://www.taxact.com
       img: taxact.png
-      tfa: Yes
-      email: Yes
-      sms: Yes
-      doc: https://www.taxact.com/support/24138/2016/how-is-a-verification-code-used
+      bch: No
+
+    - name: Tradingview
```
Comment (kenman345) [inline _data/finance.yml]: This seems like a cryptoservice, not a finance one per se. Also, I think they do accept BCH and other cryptos, but please verify when moving

**Comment by kenman345** [comment]
@joshmcintyre Please look at my two comments on your additions to `finance.yml` and make those moves/changes. Then we can merge this in.

File: _data/finance.yml (hunk 15 above)
Comment (kenman345) [reply _data/finance.yml]: Might actually be a cryptoexchange.

**Comment by joshmcintyre** [comment]
Pushed a commit to move those items from finance to the proper enabled categories. I haven't had a chance to set up a Ruby env yet to run the bundler tests before pushing, so hopefully the Travis tests come back clean.

**Comment by joshmcintyre** [comment]
Fixed up my changes to use true/false instead of yes/no

File: _data/cryptoexchanges.yml [hunk 16]
```diff
@@ -453,6 +453,15 @@ websites:
...
   othercrypto: true
+- name: Tradingview
+  url: https://www.tradingview.com/
+  img: tradingview.png
+  twitter: tradingview
+  facebook: tradingview
+  bch: false
+  btc: true
+  othercrypto: false
+ doc: https://www.tradingview.com/wiki/FAQ
```
Comment (kenman345) [inline _data/cryptoexchanges.yml]: Spacing might need to be fixed. They should be aligned even for the last tag

**Pull request merged**